hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java

```diff
@@ -522,6 +536,51 @@ public boolean serverHasTooFewRegions(int server) {
       return numRegions < minLoad;
     }
 
+    /**
+     * Return the min skew of distribution
+     */
+    public static double getMinSkew(double total, double numServers) {
```

Comment:
These methods are public because used outside this class? If not, keep them private to the class I'd say.

Reply:
Yes. They are used else where so that they are also static.